### PR TITLE
[Feature] Clean settings view to try and stabilize timeout in CI, Update API for CallCompositeAVMode to CallCompositeAudioVideoMode

### PR DESCRIPTION
**Pull request by ahammer**
## Purpose

* Clean settings view to try and stabilize timeout in CI
* Update API for CallCompositeAVMode to CallCompositeAudioVideoMode

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Pull request checklist

This PR has considered:
- [ ] Color Theming
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] View Data Injection
- [ ] Demo App UI/UX update

## How to Test

* Open...
* Click on...

## What to Check

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### CODE REVIEW

**Review by pavelprystinka — APPROVED** [review]

**Review by pavelprystinka — APPROVED** [review]

**Review by jamchengms — DISMISSED** [review]

File: AzureCommunicationUI/AzureCommunicationUIDemoApp/Sources/Views/CallingDemoView.swift [hunk 1]
```diff
@@ -32,22 +32,18 @@ struct CallingDemoView: View {
...
-            // This HSTACK if for testing toggles
-            // Loading the settings, scrolling, waiting, flaky
-            // This is easier for automation
+            // This HStack is for testing toggles.
+            // Adjusted to make buttons invisible but still accessible for automation.
             HStack {
                 Button("AudioOnly") {
                     envConfigSubject.audioOnly = !envConfigSubject.audioOnly
                 }
-                .background(envConfigSubject.audioOnly ? Color.green : Color.red)
```
Comment (jamchengms) [inline AzureCommunicationUI/AzureCommunicationUIDemoApp/Sources/Views/CallingDemoView.swift:42]: why are we removing these?

File: AzureCommunicationUI/Podfile.lock [hunk 2]
```diff
@@ -84,4 +84,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 78ea59a451b84782ef840c80dc01e9afa5adf125
 
-COCOAPODS: 1.15.0
+COCOAPODS: 1.14.3
```
Comment (jamchengms) [inline AzureCommunicationUI/Podfile.lock:87]: do we need this change?

File: AzureCommunicationUI/AzureCommunicationUIDemoApp/Sources/Views/CallingDemoView.swift (hunk 1 above)
Comment (ahammer) [reply AzureCommunicationUI/AzureCommunicationUIDemoApp/Sources/Views/CallingDemoView.swift:42]: They are part of the "hidden" buttons for Automation to use, they had some color (showed as lines) but it's not necessary, so I took away the color.

File: AzureCommunicationUI/Podfile.lock (hunk 2 above)
Comment (ahammer) [reply AzureCommunicationUI/Podfile.lock:87]: need, I'm not sure, but this is the result of a podfile.lock conflict and running pod update. It's automatic and not by choice.

Comment (ahammer) [reply AzureCommunicationUI/Podfile.lock:87]: however, security does ask if this kind of stuff is all up to date, so it probably should be updated.

It's not the podspec, and not related to the publication.

Comment (pavelprystinka) [reply AzureCommunicationUI/Podfile.lock:87]: It is just a lock file. It is re-generated locally every time we do "pod install".
Version is just what was a local tool installed.

**Review by pavelprystinka — DISMISSED** [review]

File: AzureCommunicationUI/Podfile.lock (hunk 2 above)
Comment (ahammer) [reply AzureCommunicationUI/Podfile.lock]: 15.2 now

**Review by jamchengms — APPROVED** [review]

**Review by pavelprystinka — APPROVED** [review]

**Pull request merged**